jdbc/spring-cloud-starter-stream-source-jdbc/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -1,3 +1,4 @@
 configuration-properties.classes=org.springframework.cloud.stream.app.jdbc.source.JdbcSourceProperties, \
-  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultOne, \
+  org.springframework.boot.autoconfigure.jdbc.DataSourceProperties
```

Comment:
Actually, I've got the same issue.
Seems to stem from the way boot treats those props. Most of them actually come from specific pool implementations.

BUT, there is this for example in the boot file:
```
    {
      "name": "spring.datasource.driver-class-name",
      "type": "java.lang.String",
      "description": "Fully qualified name of the JDBC driver. Auto-detected based on the URL by default.",
      "sourceType": "org.springframework.boot.autoconfigure.jdbc.DataSourceProperties"
    },
```

Weird, will require some debugging...